From 3801bba822bf88c202a348d3e01dd493c0ca0551 Mon Sep 17 00:00:00 2001
From: Nadim Asaduzzaman <nadim@nadimasad.cc>
Date: Fri, 18 Nov 2022 04:34:38 -0700
Subject: [PATCH] skia: fix compilation with MSVC2022

compilation with MSVC2022 failed due to missing string definition within std namespace

Fixes: QTBUG-108532
Change-Id: I2119952d5809895e2511ce1c2c262022af3ba191
Reviewed-on: https://codereview.qt-project.org/c/qt/qtwebengine-chromium/+/444132
Reviewed-by: Allan Sandfeld Jensen <allan.jensen@qt.io>
---
 chromium/third_party/skia/src/core/SkShaderCodeDictionary.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chromium/third_party/skia/src/core/SkShaderCodeDictionary.h b/chromium/third_party/skia/src/core/SkShaderCodeDictionary.h
index 3a54bbe22e7..8612a469862 100644
--- a/chromium/third_party/skia/src/core/SkShaderCodeDictionary.h
+++ b/chromium/third_party/skia/src/core/SkShaderCodeDictionary.h
@@ -9,6 +9,7 @@
 #define SkShaderCodeDictionary_DEFINED
 
 #include <array>
+#include <string>
 #include <unordered_map>
 #include <vector>
 #include "include/core/SkSpan.h"
